src/main/java/com/hackclub/hccore/slack/SlackBot.java

```diff
@@ -54,14 +65,22 @@ public class SlackBot implements Listener {
   private final SocketModeApp socket;
   private final String commandBase;
 
-  public static final String playerDeathMessageAvatarUrl = "https://cloud-4zgvoofbx-hack-club-bot.vercel.app/0image.png";
-  public static final String playerAfkEnterAvatarUrl = "https://cloud-pt6yc0dyx-hack-club-bot.vercel.app/0hc-afk-icon.png";
-  public static final String playerAfkLeaveAvatarUrl = "https://cloud-pt6yc0dyx-hack-club-bot.vercel.app/0hc-afk-icon.png";
-  public static final String serverConsoleAvatarUrl = "https://cloud-6lujjsrt6-hack-club-bot.vercel.app/0console_edited.png";
-  public static final String serverAvatarLink = "https://assets.hackclub.com/icon-progress-square.png";
-  public static final String playerServerLeaveAvatarUrl = "https://cloud-if9tepzbn-hack-club-bot.vercel.app/0hccoreleave.png";
-  public static final String playerServerJoinAvatarUrl = "https://cloud-if9tepzbn-hack-club-bot.vercel.app/1hccorejoin.png";
-  public static final String playerAdvancementAvatarUrl = "https://cloud-obk2f29h4-hack-club-bot.vercel.app/0achievement.png";
+  public static final String playerDeathMessageAvatarUrl =
```

Comment:
I would prefer it if all of these strings were fixed to be 1 line again.
What did you reformat it with? The bundled idea config didn't do it for me iirc